fog/ingest/client/src/main.rs

```diff
@@ -54,12 +56,14 @@ fn main() -> ClientResult<()> {
 fn get_status(logger: &Logger, ingest_client: &FogIngestGrpcClient) -> ClientResult<()> {
     let status = ingest_client.get_status().expect("rpc failed");
     log::info!(logger, "Status: {:?}", status);
+    println!("{}", ingest_summary_to_json(&status));
```

Comment:
What's the reason for the printlns? Is this read by the conformance test?

Reply:
Yup, this allows scripts to pick up the output and parse it.

Reply:
I see - neat :)